common/OpTestBMC.py

```diff
@@ -32,8 +32,11 @@
 
 import sys
 import time
-import pxssh
 import pexpect
+try:
+    import pxssh
```

Comment:
wondering why you are seeing this error.

If you are using python2.7 then you shouldn't have this issue